packages/manager/modules/nasha/src/dashboard/partitions/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,20 @@
+{
+  "nasha_dashboard_partitions_action_access": "Gérer les accès",
+  "nasha_dashboard_partitions_action_delete": "Supprimer",
+  "nasha_dashboard_partitions_action_edit_size": "Modifier la taille",
+  "nasha_dashboard_partitions_action_show": "Éditer / Voir",
+  "nasha_dashboard_partitions_action_snapshots": "Gérer les snapshots",
+  "nasha_dashboard_partitions_action_zfs_options": "Paramètres Z File System (ZFS)",
+  "nasha_dashboard_partitions_count_title": "Nombre de volumes",
+  "nasha_dashboard_partitions_count": "{{ count }}/{{ max }} (maximum)",
+  "nasha_dashboard_partitions_create": "Créer une partition",
+  "nasha_dashboard_partitions_list_description": "Description",
```

Comment:
This can be maintained as constant.

Reply:
This is a french translation actually :smiley: